### PR TITLE
Add all missing error codes

### PR DESCRIPTION
**Pull request by remi-stripe**
Fixes https://github.com/stripe/stripe-go/issues/896

r? @ob-stripe 
cc @stripe/api-libraries

### CODE REVIEW

File: error.go [hunk 1]
```diff
@@ -21,20 +21,100 @@ type ErrorCode string
...
+	ErrorCodeTLSVersionUnsupported                  ErrorCode = "tls_version_unsupported"
+	ErrorCodeTokenAlreadyUsed                       ErrorCode = "token_already_used"
+	ErrorCodeTokenInUse                             ErrorCode = "token_in_use"
+	ErrorCodeTransfersNotAllowed                    ErrorCode = "transfers_not_allowed"
+	ErrorCodeUpstreamOrderCreationFailed            ErrorCode = "upstream_order_creation_failed"
+	ErrorCodeURLInvalid                             ErrorCode = "url_invalid"
+
+	// The following error codes can be returned though are undocumented
+	ErrorCodeIncorrectCVC     ErrorCode = "incorrect_cvc"
+	ErrorCodeInvalidCVC       ErrorCode = "invalid_cvc"
```
Comment (ob-stripe) [inline error.go]: I have no idea how this is even working, but both `ErrorCodeIncorrectCVC` and `ErrorCodeInvalidCVC` are already defined (with the same values) above.

**Comment by remi-stripe** [comment]
Re-assigning to @brandur-stripe as we never finished that one

File: error.go [hunk 2]
```diff
@@ -21,20 +21,98 @@ type ErrorCode string
...
+	ErrorCodePlatformAPIKeyExpired                  ErrorCode = "platform_api_key_expired"
+	ErrorCodePostalCodeInvalid                      ErrorCode = "postal_code_invalid"
+	ErrorCodeProcessingError                        ErrorCode = "processing_error"
+	ErrorCodeProductInactive                        ErrorCode = "product_inactive"
+	ErrorCodeRateLimit                              ErrorCode = "rate_limit"
+	ErrorCodeResourceAlreadyExists                  ErrorCode = "resource_already_exists"
+	ErrorCodeResourceMissing                        ErrorCode = "resource_missing"
+	ErrorCodeRoutingNumberInvalid                   ErrorCode = "routing_number_invalid"
+	ErrorCodeSecretKeyRequired                      ErrorCode = "secret_key_required"
+	ErrorCodeSepaUnsupportedAccount                 ErrorCode = "sepa_unsupported_account"
```
Comment (brandur-stripe) [inline error.go]: Last one Remi: I think we want to go with "SEPA" instead of "Sepa" here.

**Comment by brandur-stripe** [comment]
One more casing issue, but other than that LGTM! Love how comprehensive this list is.

ptal @remi-stripe

File: error.go (hunk 2 above)
Comment (remi-stripe) [reply error.go]: We use `Sepa` everywhere else in the library though for SepaDebit. I don't think we should change that one but happy to make the change if you prefer on that one

Comment (brandur-stripe) [reply error.go]: Ah, k. I looked around online and I think "SEPA" is definitely the overall preferred casing (and use in Stripe's docs, etc. too).

That said, given we have pre-existing `Sepa` in stripe-go, I'm good just keeping it like this for consistency.

Comment (remi-stripe) [reply error.go]: Yeah, I'm unsure on that one, I think both would make sense so deferring to you but happy to make the change

Comment (brandur-stripe) [reply error.go]: K cool. Easiest to go with "Sepa" for now then, so just leave it. Thanks!

**Comment by brandur-stripe** [comment]
Released as 61.20.0.

**Comment by layby42** [comment]
`invoice_payment_intent_requires_action`  (?)

**Comment by remi-stripe** [comment]
@layby42 Would you have a bit more details than just this code? It's not listed in https://stripe.com/docs/error-codes today which is why it wasn't added. I'd recommend flagging to our support team if it's a code you rely on: https://support.stripe.com/contact

**Comment by layby42** [comment]
@remi-stripe happens when you try to pay invoice (Invoices.Pay) for "incomplete" subscription with card payment method (stripe.InvoicePayParams -> PaymentMethod) that requires 3D Secure authorization on attempt to charge. 

Also (not directly related) would be nice to get Invoice from stripe.Error (the way it is possible to get PaymentIntent or PaymentMethod) to not re-request if from Stripe.

**Comment by remi-stripe** [comment]
> Also (not directly related) would be nice to get Invoice from stripe.Error (the way it is possible to get PaymentIntent or PaymentMethod) to not re-request if from Stripe.
Yeah we've been thinking about this internally too but no short term plan to support this unfortunately.

**Pull request merged**